wp-multi-network/includes/functions.php

```diff
@@ -592,7 +592,7 @@ function add_network( $args = array() ) {
 			: get_site_option( 'ms_files_rewriting' );
 
 		// Not using rewriting, and using a newer version of WordPress than 3.7.
-		if ( empty( $use_files_rewriting ) && version_compare( $wp_version, '3.7', '>' ) ) {
+		if ( empty( $use_files_rewriting ) && version_compare( $wp_version, '3.7', '<' ) ) {
```

Comment:
@JJJ The new logic is "Not using rewriting, and using an _older_ version of WordPress than 3.7." This results in a new network using `blogs.dir` since `upload_path` and `upload_url_path` are empty.

<img width="384" alt="Screenshot 2021-09-17 at 16 48 21" src="https://user-images.githubusercontent.com/726936/133817130-71398560-373b-44fb-af3a-f9500b4a0434.png">

Reply:
@christianwach Reverted the revert 😅

Reply:
> Reverted the revert 😅

@JJJ 🆒  That still leaves the `/sites/*/sites/*` conundrum and what to do about it. PR as it stands still produces:

<img width="362" alt="Screenshot 2021-09-18 at 12 11 04" src="https://user-images.githubusercontent.com/726936/133886807-0756dc43-2693-4d5b-8f97-f3436d2dd667.png">

[As I said](https://github.com/stuttter/wp-multi-network/issues/153#issuecomment-920176155), for new multi-network installs it seems sensible [not to append `$ms_dir`](https://github.com/stuttter/wp-multi-network/blob/5603be73b253cac0789c1a0632689efb0d18f616/wp-multi-network/includes/functions.php#L610-L617) but for existing sites that have that schema? I'm not sure.